addons/sale/static/src/js/product_configurator_mixin.js

```diff
@@ -64,19 +64,31 @@ var ProductConfiguratorMixin = {
         var qty = $component.find('input[name="add_qty"]').val();
 
         var $parent = $(ev.target).closest('.js_product');
+        var uniqueId = self._getUniqueId($parent);
+
+        // Guarantee to show the info for the last selected combination only.
+        // Needs to be saved per uniqueId because on the configurator modal
+        // there are several elements potentially calling this method.
+        this.dropPrevious = this.dropPrevious || {};
+        this.dropPrevious[uniqueId] = this.dropPrevious[uniqueId] || new concurrency.DropPrevious();
```

Comment:
These two lines should be in the function that is memiozed and the correct dropPrevious can be automatically given as second argument of this function. This would also remove the need of the `_getUniqueID` method.

Reply:
`_getUniqueID` is also called to memoize in the first place, so we still need it I think

Reply:
@seb-odoo Strange... I replied to this yesterday... :/

-> My point was that the function is not needed anymore as only called at one place and there is no point extending it (so just include the code at the only place it will be used).

You can do that then we can check whatever there is to check together :)

Reply:
Done moving it into the memoized, it looks better now.
I also used DropMisordered instead because more suited to this case.